doc/federation.md

```diff
@@ -0,0 +1,83 @@
+# Federation Management
+
+This document describes the Federation Management of Tapyrus.
+
+## Overview
+
+The Tapyrus Signer Network is a permissioned network consisting of identifiable signers who can control to sign blocks.
+These signers can not only sign blocks but also can control the identity of the participants on the network themselves.
+We introduce Federation which is consisted of a set of identifiable signers and the features to maintain their access controls.
+
+A new signer joining the network needs to get approval from the signers who belongs to the federation.
+Federation including a new signer needs to recreate a new aggregate public key and secret in the same protocol as [How to set up new Tapyrus Signer Network](./setup.md).
+The created aggregated public key is committed by the signer in existing federation and submitted to TapyrusCore with the block.
+A block has a field for a new aggregated public key.
+When a new aggregated public key is set in a block, TapyrusCore discards the public key that has been used so far, and performs proof verification using the new aggregate public key from the next block.
+When leaving from federation, it is necessary to update the aggregated public key in the same protocol.
+
+## How To Update Federation
+
+The following describes how to change the federation.
+
+### Re-create aggregated public key and secret.
+
+Federation members, including the new signer, regenerate the aggregated public key and the secret in the same procedure as Step 2-Step 4 in [How to set up new Tapyrus Signer Network](./setup.md).
+Note that the same static public key can be used for the existing signer, so Step1 is not required. However, for a new signer, a key pair needs to be created in Step 1.
```

Comment:
Step 1 is "Choose network ID", so Step 2 is correct?

Reply:
Yes, I'll fix.

Reply:
fix at 59416cc